### DISCUSSION TITLE
Red bar for current time behind events?

### DISCUSSION THREAD

**Discussion by joe-claire** [niccokunzmann/open-web-calendar#465] [Q&A]
Is there any way to make the red bar that shows where the current time is in a schedule appear overlaying on top of the current event? I believe as of right now it is behind event blocks. So the red bar is present when nothing is scheduled, but is gone when a currently scheduled event is in progress.

**Comment by niccokunzmann** [comment]
Hi, I converted this to a discussion and we could make it a feature request for an option.

What I would do is modify the CSS's [z-index](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index):

```css
.dhx_now_time{
   z-index: 10;
}
```

I hope that 10 is enough...
